### DISCUSSION TITLE
Welcome to OmniBOR Discussions!

### DISCUSSION THREAD

**Discussion by edwarnicke** [omnibor/spec#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alilleybrinker** [comment]
Hi everyone! My name is Andrew. I'm a contributor to the OmniBOR spec, and to the Rust implementation on OmniBOR. My current goal is to get the Rust implementation into a place where it is usable as the basis (via FFI) for implementations in other languages.
